spark.js

```diff
@@ -195,12 +164,6 @@ Spark.readable('__initialise', [function initialise() {
   // We've received new data from our client, decode and emit it.
   //
   ultron.on('incoming::data', function message(raw) {
-    //
```

Comment:
Re: heartbeats firing only when idle, the documentation was correct; here it was.